django/applications/catmaid/tests/apis/test_labels.py

```diff
@@ -106,3 +106,28 @@ def test_label_cardinality_warning(self):
         self.assertEqual(response.status_code, 200)
         self.assertTrue('warning' in parsed_response)
         self.assertTrue('soma (2, max. 1)' in parsed_response['warning'])
+
+
+    def test_stats(self):
+        self.fake_authentication()
+        url = '/{}/labels/stats'.format(self.test_project_id)
+        response = self.client.get(url)
+
+        self.assertEqual(response.status_code, 200)
+
```

Comment:
I suspect that this means that every node of a skeleton has a label on it with the ID/ name of the skeleton - is this expected/wanted? This behaviour shows up in the '1.5 abdominal segments of the Drosophila VNC' project but not the 'atonal' project. This is probably a defect.